core/src/main/kotlin/fr/sncf/osrd/api/api_v2/pathfinding/PathfindingPostProcessing.kt

```diff
@@ -0,0 +1,91 @@
+package fr.sncf.osrd.api.api_v2.pathfinding
+
+import fr.sncf.osrd.api.FullInfra
+import fr.sncf.osrd.api.pathfinding.makeRoutePath
+import fr.sncf.osrd.graph.Pathfinding
+import fr.sncf.osrd.graph.PathfindingEdgeLocationId
+import fr.sncf.osrd.graph.PathfindingResultId
+import fr.sncf.osrd.railjson.schema.infra.RJSRoutePath
+import fr.sncf.osrd.sim_infra.api.Block
+import fr.sncf.osrd.sim_infra.api.BlockId
+import fr.sncf.osrd.sim_infra.api.Path
+import fr.sncf.osrd.utils.indexing.StaticIdx
+import fr.sncf.osrd.utils.units.Length
+import fr.sncf.osrd.utils.units.Offset
+import fr.sncf.osrd.utils.units.Offset.Companion.max
+import fr.sncf.osrd.utils.units.Offset.Companion.min
+import fr.sncf.osrd.utils.units.meters
+import fr.sncf.osrd.utils.units.sumDistances
+
+fun runPathfindingPostProcessing(
+    infra: FullInfra,
+    rawPath: PathfindingResultId<Block>
+): PathfindingBlockResponse {
+    // We reuse some of the old function of pathfindingResultConverter,
+    // there will be some cleanup to be made when the old version is removed
+    val oldRoutePath = makeRoutePath(infra.blockInfra, infra.rawInfra, rawPath.ranges)
```

Comment:
I've had a look at the existing code, and I think it can be pretty easily copied over and adapted